suites/apis/OAuth2Test.js

```diff
@@ -24,21 +25,21 @@ Scenario('Authorization code flow: Test that successfully generates code', async
     fence.props.clients.client.id, 'code', 'openid+user',
   );
   fence.ask.assertContainSubStr(resULR, ['code=']);
-});
+}).retry(2);
```

Comment:
oh :man_facepalming: this is the retry I was talking about in a previous comment. :+1:

Reply:
Correct.
Once we apply the mitigation as the first call to action, we should try to increase the observability around such retries and deliberate further on whether or not we want to keep them or:
- Perform further low level investigation on why the system produces invalid results intermittently.
- Make the tests smarter by avoiding garbage from previous runs that haven't been cleaned-up properly,  some smart sleep calls to eliminate potential race conditions, etc.